charts/datasaur/templates/config/datasaur-env-configMap.yaml

```diff
@@ -12,6 +12,18 @@ data:
   LANGUAGE_TOOL_SERVICE_ENDPOINT: http://{{ .Release.Name }}-language-tool-service:8010/v2/check
   PRESIDIO_ANALYZER_ENDPOINT: http://{{ .Release.Name }}-presidio-analyzer-service:3000/analyze
   WAVEFORM_PEAKS_GENERATOR_HOST: http://{{ .Release.Name }}-audiowaveform-generator-service:5005
+  URL_INSPECTOR_INGRESS_URL: https://{{ .Values.ingress.host }}
```

Comment:
can we set the host to different domain? like i explained previously, the upload proxy should have different url

datasaur app -> cloudflare proxy on -> file size limitation
upload proxy -> cloudflare proxy off -> no file size limitation 

@daniel-jeremia

Reply:
should this be an option or upload-proxy must have different url? @tonoman3g

Reply:
lets continue it on another PR